src/app/components/LanguageNameInput.svelte

```diff
@@ -103,6 +77,6 @@
     labelFieldName="language" />
   <p class="autocomplete__subtext" data-cy="autocomplete-subtext">
     BCP47Tag:
-    {subtext || ''}
+    {selectedLanguage?.bcp47Tag ?? ''}
```

Comment:
Nice use of the null-coalescing operator!

Reply:
I have to thank @yehee for that one 😅